Formula/m/minidjvu.rb

```diff
@@ -3,7 +3,7 @@ class Minidjvu < Formula
   homepage "https://minidjvu.sourceforge.net/"
   url "https://downloads.sourceforge.net/project/minidjvu/minidjvu/0.8/minidjvu-0.8.tar.gz"
   sha256 "e9c892e0272ee4e560eaa2dbd16b40719b9797a1fa2749efeb6622f388dfb74a"
-  license "GPL-2.0"
+  license "GPL-2.0-only"
```

Comment:
`docs/minidjvu.1`:

```
.\" Copyright (c) 2005  Ilya Mezhirov
.\" Copyright (c) 2009  Alexey Kryukov
.\"
.\" This is free documentation; you can redistribute it and/or
.\" modify it under the terms of the GNU General Public License as
.\" published by the Free Software Foundation; either version 2 of
.\" the License, or (at your option) any later version.
.\"
.\" The GNU General Public License's references to "object code"
.\" and "executables" are to be interpreted as the output of any
.\" document formatting or typesetting system, including
.\" intermediate and printed output.
.\"
.\" This manual is distributed in the hope that it will be useful,
.\" but WITHOUT ANY WARRANTY; without even the implied warranty of
.\" MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
.\" GNU General Public License for more details.
.\"
.\" You should have received a copy of the GNU General Public
.\" License along with this manual. Otherwise check the web site
.\" of the Free Software Foundation at http://www.fsf.org.
.\"
```

```suggestion
  license "GPL-2.0-or-later"
```